mod.ts

```diff
@@ -66,13 +66,20 @@ namespace Drash {
     export const HttpResponseException = http_response_exception;
   }
 
-  export namespace Loggers {
+  export namespace CoreLoggers {
     export type ConsoleLogger = console_logger;
     export const ConsoleLogger = console_logger;
     export type FileLogger = file_logger;
     export const FileLogger = file_logger;
     export type Logger = base_logger;
     export const Logger = base_logger;
+  };
+
+  export const Loggers: any = {};
+
+  export function addLogger(name: string, logger: any) {
+    // todo: throw error
```

Comment:
with todo statments, i go with the what deno is doing as a standard. example:

```
TODO(crookse) throw error
```

Reply:
im not saying you have to conform to it, but it's nice to know who wrote it in case it's a long-running TODO

Reply:
Good point. I'll keep that in mind.